### PR TITLE
51 todo setting up todo list canvas

### LINKED ISSUE #51: [Todo] Setting up Todo List canvas
# [Todo] Setting up Todo List canvas

### Objectives
Set up the HTML page for the Todo List canvas
### Acceptance Criteria
- [x] Set up the 3 columns with a few dummy tasks hard coded
- [x] Set up page title
- [x] Include the dashboard headers with navigation to other pages
### Assignees
- @shjucsd 
- @UKCSD 
- @alt012ucsd 
- @ChooChooChao

### PR DESCRIPTION
**Pull request by ChooChooChao**
Objectives
- Set up the HTML page for the Todo List canvas

Acceptance Criteria
- [x ] Set up the 3 columns with a few dummy tasks hard coded
- [x ] Set up page title
- [x ] Include the dashboard headers with navigation to other pages

### CODE REVIEW

**Review by shjucsd — CHANGES_REQUESTED** [review]
The css looks good. For the hard-coded data, can we have the number of tasks display for in progress and done as well just so that we are consistent with the ui? Similarly, can you add the tags for different labels like Work, Personal.

**Review by alt012ucsd — APPROVED** [review]
Everything looks good, all base functionality accounted for.

**Review by shjucsd — APPROVED** [review]
I added the functionality for updating the localStorage on adding and deleting the tasks and dynamically updating the display count as well, i also updated the formatting using prettier

File: to_do/to-do-app.js [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+
+  const toDoTemplate = {
+    id: Math.floor(Math.random() * 2000000),
+    title: "Default Title",
+    dueDate: `${currentDate}`,
+    label: "self",
+  };
+
+  if (existing) {
+    console.log("existing");
```
Comment (adironene) [inline to_do/to-do-app.js]: if we are logging stuff in console, I would make it more detailed. Like existing what? + more context

File: to_do/to-do.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                <div id="done-list">
+                    <list>
+                        <div class="task">
+                            <input type="checkbox" id="radio-button"> 
+                            <p>Journal Canvas</p>
+                            <div class="category">
+                                <p id="work-category">work</p>
+                            </div>
+                            <span id="due-date">Due Date: 5/28</span>
+                            <!-- <div class="buttons">
```
Comment (adironene) [inline to_do/to-do.html]: remove unused comments

File: to_do/to-do-app.js [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+  );
+  console.log("clicked addbutton in progress");
+
+  // Add saved tasks for Not Started and In Progress from localStorage
+  addTask(toDoContainer, true);
+
+  updateCount();
+}
+
+function addTask(toDoContainer, existing, category) {
```
Comment (adironene) [inline to_do/to-do-app.js]: I would add a bit more comments throughout the doc to tell the reader what the code is doing. It's self-explanatory after reading through a few times but having comments before each "group" of tasks would be very helpful!

**Review by adironene — CHANGES_REQUESTED** [review]
Amazing work! Good job dividing the tasks up in different functions. Just left a few comments for suggestions and good practice.

Also **todo** - please attach a video for UI demo

**Review by kabirsachdev7 — APPROVED** [review]
Foundation for the backend is good. Definitely meets the acceptance criteria. Phoebe touched on some points already, but just want to say to add more comments pls wherever you can as you continue building the todo.

**Comment by shjucsd** [comment]
for the logging into the console comment, i think we just had those to check our implementation before and that the code was entering the correct places, we can remove them and i can add comments for all the functions so that they can be used to create jsdocs

**Review by adironene — APPROVED** [review]
Fixes look good!

**Pull request merged**